### PR TITLE
LOINC imports

### PR DESCRIPTION
**Pull request by turbomam**

### CODE REVIEW

**Comment by turbomam** [comment]
@jamesaoverton I apologize for pestering. I was able to find some errors I made in my submission, but I am still getting the following and don't understand. I ran `make all` on my laptop and didn't see any complaints.

```Bash
java -jar build/robot.jar --prefix "REO: http://purl.obolibrary.org/obo/REO_" merge \
--input src/ontology/obi-edit.owl \
template \
--template src/ontology/templates/sequence-analysis.tsv \
annotate \
--ontology-iri "http://purl.obolibrary.org/obo/obi/dev/modules/sequence-analysis.owl" \
--output src/ontology/modules/sequence-analysis.owl
Could not load imported ontology: <http://purl.obolibrary.org/obo/ro/annotations.owl> Cause: connect timed out
```

File: src/ontology/OntoFox_inputs/OBO_input.txt [hunk 1]
```diff
@@ -157,6 +157,44 @@ http://purl.obolibrary.org/obo/PR_000025475
 http://purl.obolibrary.org/obo/PR_000025477
 http://purl.obolibrary.org/obo/PR_000025478
 http://purl.obolibrary.org/obo/PR_000003252
+http://purl.obolibrary.org/obo/DRON_00018971 # adalimumab for TURBO assays
```
Comment (jamesaoverton) [inline src/ontology/OntoFox_inputs/OBO_input.txt:160]: This seems to be out of place.

File: src/ontology/OntoFox_inputs/NCBITaxon_input.txt [hunk 2]
```diff
@@ -34,6 +34,18 @@ http://purl.obolibrary.org/obo/NCBITaxon_11103
 http://purl.obolibrary.org/obo/NCBITaxon_10376
 http://purl.obolibrary.org/obo/NCBITaxon_10358
 http://purl.obolibrary.org/obo/NCBITaxon_160
+http://purl.obolibrary.org/obo/NCBITaxon_1 # root for TURBO assays
```
Comment (jamesaoverton) [inline src/ontology/OntoFox_inputs/NCBITaxon_input.txt:37]: Can you use OBI 'organism' instead? That's what we've done elsewhere.

File: src/ontology/OntoFox_inputs/UO_input.txt [hunk 3]
```diff
@@ -16,6 +16,9 @@ http://purl.obolibrary.org/obo/UO_0000105
 http://purl.obolibrary.org/obo/UO_0000109
 http://purl.obolibrary.org/obo/UO_0000270
 http://purl.obolibrary.org/obo/UO_0000280
+http://purl.obolibrary.org/obo/UO_0000061 # unit of molarity for TURBO assays
```
Comment (jamesaoverton) [inline src/ontology/OntoFox_inputs/UO_input.txt:19]: These three should be in just one of these UO files. It just depends on whether we want it in OBI as a class or as an instance.

**Comment by jamesaoverton** [comment]
I made some specific comments. One general comment: the various "release" files should not be included in the PR. This means `obi.owl` and the files in `views/`. I just update these when we make a new release. You only need to commit stuff under `src/`. This is probably not clear in the REAMDE...

**Comment by jamesaoverton** [comment]
This is a (temporary) network error "Could not load imported ontology: <http://purl.obolibrary.org/obo/ro/annotations.owl> Cause: connect timed out". When I see one of these, I click the "Restart build" button.

**Comment by jamesaoverton** [comment]
Is this PR supposed to add one term for #1202, or is it supposed to update hundreds or imports for the LOINC work?

**Comment by turbomam** [comment]
Well, probably both. Is there something I can do to split those two things
up?

**Comment by jamesaoverton** [comment]
I moved the work on #1202 to new PR #1212.

To update imports you need to run `make imports`. Then the updated files in `src/ontology/OntoFox_outputs` should be added to this PR.

However it will not be that simple. Imports go out of date as the upstream ontologies change. When I ran this locally and checked the diffs, it looked like GO had obsoleted some terms OBI was using, for starters. I'm also surprised that you need *so* many ChEBI terms, and I have other questions about the new imports.

Overall, it might be better to update the update the imports, one-by-one in separate PRs, without your newly imported terms, and resolve any problems with the upstream ontologies. Then it would be easier to import your new terms.

I'm sorry that this is difficult. There's about a thousand moving pieces.

**Comment by jamesaoverton** [comment]
Since this is not ready to go, I converted to a draft to prevent an accidental merge.

**Comment by jamesaoverton** [comment]
I updated the import files (without adding any new imported terms) in #1216, and I updated this branch with all the latest changes. So you can continue your LOINC work on this branch.

**Comment by jamesaoverton** [comment]
@turbomam What's the status of this? Should we mark it as Draft again?

**Pull request closed without merging**